[2022-08-29T19:26:10.368-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2022-08-13T00:00:00+00:00 [queued]>
[2022-08-29T19:26:10.384-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2022-08-13T00:00:00+00:00 [queued]>
[2022-08-29T19:26:10.384-0400] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T19:26:10.385-0400] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-08-29T19:26:10.385-0400] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T19:26:10.423-0400] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): task1> on 2022-08-13 00:00:00+00:00
[2022-08-29T19:26:10.426-0400] {standard_task_runner.py:52} INFO - Started process 10132 to run task
[2022-08-29T19:26:10.431-0400] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_catchup_backfill_v02', 'task1', 'backfill__2022-08-13T00:00:00+00:00', '--job-id', '32', '--pool', 'default_pool', '--raw', '--subdir', '/home/augenz/PycharmProjects/airflow/dags/dag_with_catchup_and_backfill_v02.py', '--cfg-path', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmp0kv_2eje', '--error-file', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmpfpfcaa4v']
[2022-08-29T19:26:10.432-0400] {standard_task_runner.py:80} INFO - Job 32: Subtask task1
[2022-08-29T19:26:10.621-0400] {task_command.py:371} INFO - Running <TaskInstance: dag_with_catchup_backfill_v02.task1 backfill__2022-08-13T00:00:00+00:00 [running]> on host pop-os.localdomain
[2022-08-29T19:26:10.890-0400] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=dag_with_catchup_backfill_v02
AIRFLOW_CTX_TASK_ID=task1
AIRFLOW_CTX_EXECUTION_DATE=2022-08-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=backfill__2022-08-13T00:00:00+00:00
[2022-08-29T19:26:10.893-0400] {subprocess.py:62} INFO - Tmp dir root location: 
 /run/user/1000/app/com.jetbrains.PyCharm-Community
[2022-08-29T19:26:10.894-0400] {subprocess.py:74} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2022-08-29T19:26:10.906-0400] {subprocess.py:85} INFO - Output:
[2022-08-29T19:26:10.912-0400] {subprocess.py:92} INFO - This is a simple bash command!
[2022-08-29T19:26:10.912-0400] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-29T19:26:11.027-0400] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_backfill_v02, task_id=task1, execution_date=20220813T000000, start_date=20220829T232610, end_date=20220829T232611
[2022-08-29T19:26:11.085-0400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-29T19:26:11.222-0400] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-29T19:26:11.291-0400] {dagrun.py:567} INFO - Marking run <DagRun dag_with_catchup_backfill_v02 @ 2022-08-13 00:00:00+00:00: backfill__2022-08-13T00:00:00+00:00, state:running, queued_at: None. externally triggered: False> successful
[2022-08-29T19:26:11.293-0400] {dagrun.py:612} INFO - DagRun Finished: dag_id=dag_with_catchup_backfill_v02, execution_date=2022-08-13 00:00:00+00:00, run_id=backfill__2022-08-13T00:00:00+00:00, run_start_date=2022-08-29 23:25:29.835559+00:00, run_end_date=2022-08-29 23:26:11.292737+00:00, run_duration=41.457178, state=success, external_trigger=False, run_type=backfill, data_interval_start=2022-08-13 00:00:00+00:00, data_interval_end=2022-08-14 00:00:00+00:00, dag_hash=None
